example/deploy_status.js

```diff
@@ -61,6 +129,36 @@ let cmdLineArgs = yargs
     })
     .argv;
 
+if (cmdLineArgs.proxy){
+  cmdLineArgs.host = cmdLineArgs.proxy.host;
+  cmdLineArgs.port = ~~(cmdLineArgs.proxy.port);
+  cmdLineArgs.protocol = cmdLineArgs.proxy.protocol;
+}
+
+_(cmdLineArgs).get("colSize", []).forEach((o)=>{
+  _.set(cmdLineArgs.col, o[0], o[1]);
+})
+const progressBar = ((units)=>{
+  const  Progress = require('cli-progress');
+  var progressBar = new Progress.Bar({
+    format: '[{bar}] {percentage}%'
+});
+progressBar.start(100, 0);
+progressBar.finishInterval = false;
+return progressBar;
+})(100)
+let progressCounter = 0;
```

Comment:
UX: Since we're not actually providing progress information here, perhaps is it better to replace it by a spinner (e.g. with the help of a library like `cli-spinner`, or manually by checking the existence of a tty, then sending ASCII sequences to update the spinner).